splinepy/microstructure/tiles/double_lattice.py

```diff
@@ -46,19 +48,38 @@ def create_tile(
         -------
         microtile_list : list(splines)
         """
-
+        index_second_value = 1
         if not isinstance(contact_length, float):
             raise ValueError("Invalid Type")
         if not ((contact_length > 0.0) and (contact_length < 1.0)):
             raise ValueError("Contact length must be in (0.,1.)")
 
+        # Maintain backwards compatibility
+        if parameter_sensitivities is None:
+            pass
+        elif parameter_sensitivities.shape[1] == 1:
+            _warning("DoubleLattice now expects 2 values")
+            index_second_value = 0
+            self._n_info_per_eval_point = 1
+        if parameters is None:
+            pass
+        elif parameters.shape[1] == 1:
+            _warning("DoubleLattice now expects 2 values")
+            index_second_value = 0
+            self._n_info_per_eval_point = 1
+
         # set to default if nothing is given
         if parameters is None:
             self._logd("Tile request is not parametrized, setting default 0.2")
-            parameters = _np.ones((1, 1)) * 0.1
+            parameters = _np.ones((1, 2)) * 0.1
         else:
-            if not (_np.all(parameters > 0) and _np.all(parameters < 0.25)):
-                raise ValueError("The parameter must be in 0.01 and 0.25")
+            if not (
+                _np.all(parameters > 0)
+                and _np.all(parameters < 0.5 / (1 + _np.sqrt(2)))
+            ):
+                raise ValueError(
+                    "Parameters must be between 0.01 and 0.5/(1+sqrt(2))=0.207"
+                )
```

Comment:
```suggestion
        if not (
            _np.all(parameters > 0)
            and _np.all(parameters < 0.5 / (1 + _np.sqrt(2)))
        ):
            raise ValueError(
                "Parameters must be between 0.01 and 0.5/(1+sqrt(2))=0.207"
            )
```
This should not be trapped in the else condition

Reply:
The else is not required here